src/Dapi/Response/GetACHPullResponse.cs

```diff
@@ -0,0 +1,23 @@
+using Dapi.Types;
+using Newtonsoft.Json;
+
+namespace Dapi.Response {
+    public class GetACHPullResponse : BaseResponse {
+
+        /// <summary>
+        /// This is used only to automate the deserialization of the get response.
+        /// This is a private constructor to this lib.
+        /// </summary>
+        [JsonConstructor]
+        internal GetACHPullResponse(string reference, APIStatus status, bool success, string operationID) :
```

Comment:
This needs to return a `transfer` object, of type `ACHPull`, and not the `reference`. @asmsh maybe take a look here as I'm not familiar enough with Java to give a good review

Reply:
@street-jackal will be pushing the changes again.

Reply:
37b359065f9de0cdc1256cbf0c52db9fcd9ba2b2